prow/config/config.go

```diff
@@ -66,6 +67,16 @@ type Trigger struct {
 	TrustedOrg string `json:"trusted_org,omitempty"`
 }
 
+// SlackEvent is config for the slackevents plugin.
+// If a PR is pushed to any of the repos listed in the config
+// then sent message to the all the  slack channels listed if pusher is NOT in the whitelist.
+type SlackEvent struct {
+	// Repos is either of the form org/repos or just org.
+	Repos     []string `json:"repos,omitempty"`
+	Channels  []string `json:"channels,{\"sig-contribex\",\"kubernetes-dev\"}"`
```

Comment:
Sorry, I messed it while resolving conflict. Now have comments added and fixed the son field.